src/main/java/it/gov/pagopa/ranking/model/InitiativeConfig.java

```diff
@@ -30,4 +31,6 @@ public class InitiativeConfig {
     private long size;
     private List<Order> rankingFields;
     private String rankingPathFile;
+    private LocalDateTime rankingGeneratedTimeStamp;
+    private LocalDateTime rankingPublishedTimeStamp;
```

Comment:
let's also add the fields where to store the number of onboardingOk and onboardingKo